packages/unlock-js/src/web3Service.ts

```diff
@@ -401,4 +401,17 @@ export default class Web3Service extends UnlockService {
       this.providerForNetwork(network)
     )
   }
+
+  /**
+   * Returns numbers of owners for a specific lock
+   * @param {*} lockAddress
+   * @param {*} network
+   */
+  async numberOfOwners(lockAddress: string, network: number) {
+    const lockContract = await this.getLockContract(
+      lockAddress,
+      this.providerForNetwork(network)
+    )
+    return lockContract.numberOfOwners()
```

Comment:
We also need from the contacts a way to get total of active keys

Reply:
This might differ based on the lock version. Please test if it works with older versions.
Also when changing unlockJS, please add to the changelog and make sure you update the version in the package file!

Reply:
Also, make sure you convert to a `Number` here so that applications don't have to do it themselves.